test/integration/logging_test.go

```diff
@@ -0,0 +1,236 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+//go:build integration
```

Comment:
If we are adding this test in a separate file called `logging_test` I think that we should restructure the rest of the tests, maybe something like this:
- Move `TestKubeArchiveDeployments` and `TestDatabaseConnection` to its own file and call it something like `resiliency_test.go`
- Rename `integration_test.go` to `read_archived_test.go

Reply:
I'll see what I can do.

Reply:
I divided the tests into files but not by test case, even if that matches now. The idea is that each file contains a set of related tests. I left the "TestReadAndArchive" into the current file and renamed it "TestNormalOperation".